### PR TITLE
adding contributions fk object relationship to users

### PR DESCRIPTION
**Pull request by CryptoGraffe**
## Motivation and Context

for new allocation UI I need to be able to aggregate the count of contributions for each user

## Description

modifies hasura to take advantage of the existing fk relationship

### CODE REVIEW

**Comment by CryptoGraffe** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1388** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/1388" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/coordinape/coordinape/1388?utm_source=stack-comment).

File: hasura/metadata/cron_triggers.yaml [hunk 1]
```diff
@@ -1,44 +1 @@
-- name: check-nominee
```
Comment (zashton) [inline hasura/metadata/cron_triggers.yaml:1]: don't think this should be removed

**Review by zashton — CHANGES_REQUESTED** [review]

File: hasura/metadata/cron_triggers.yaml (hunk 1 above)
Comment (CryptoGraffe) [reply hasura/metadata/cron_triggers.yaml:1]: woo good catch. i had my local state in a funk from seeding issues when i did a metadata export

Comment (CryptoGraffe) [reply hasura/metadata/cron_triggers.yaml]: fixed this

**Review by zashton — APPROVED** [review]
lgtm!

**Pull request merged**